src/kite/messagescrubber.js

```diff
@@ -14,8 +19,8 @@ export default class MessageScrubber {
       kite: this.kite.getKiteInfo(),
       authentication: this.kite.options.auth,
       withArgs: params,
-      responseCallback(response = {}) {
-        const { error: rawErr, result } = response
+      responseCallback(response) {
+        const { error: rawErr, result } = response || {}
```

Comment:
What is the reason for going back to this solution? (not criticizing, just curious)